build/tools/Automation/Program.cs

```diff
@@ -95,6 +99,14 @@ public static bool DatesMatch(string dateTarget, string dateReleased)
             return match;
         }
 
+        /// <Summary>
+        /// Caches the storage account sdk versions to avoid pulling all SDKs everytime
+        /// we check if a version exists in the storage account.
+        /// </Summary>
+        /// <param name="platform">dotnet, nodejs, python, etc
+        /// This dateTarget can be passed through github actions through an argument</param>
+        public abstract Task CacheSdkVersionsAsync(string platform);
```

Comment:
nit: just for my understanding, how did we go about "pulling all SDKs" previously? I think it's good for us to read the prod storage account when the `DotNet` object is instantiated so we can freely use the list of published production SDKs in that lifespan of the object. I do wonder if we should replace `Cache` with `Pull` since the verb `Cache` leads me to believe that these SDK versions we pull down will persist across sessions so that we don't have to pull from prod next time we run the cron job.